core/core-configuration/src/main/webapp/WEB-INF/conf/wcm-core/core-services-configuration.xml

```diff
@@ -690,6 +690,53 @@
     </component-plugin>
   </external-component-plugins>
   
+  <!-- Configure User Navigation caches - Standalone mode -->
+  <external-component-plugins>
+    <target-component>org.exoplatform.services.cache.CacheService</target-component>
+    <component-plugin>
+      <name>addExoCacheConfig</name>
+      <set-method>addExoCacheConfig</set-method>
+      <type>org.exoplatform.services.cache.ExoCacheConfigPlugin</type>
+      <description>Configures the cache for User Navigation</description>
+      <init-params>
+        <object-param>
+          <name>CacheUserNavigationKeeper</name>
+          <description></description>
+          <object type="org.exoplatform.services.cache.ExoCacheConfig">
+            <field name="name"><string>CacheUserNavigationKeeper</string></field>
+            <field name="maxSize"><int>${wcm.cache.userNavigation.capacity:300}</int></field>
+            <field name="liveTime"><long>${wcm.cache.userNavigation.timetolive:600000}</long></field>

```

Comment:
The time to live could be equal to the HTTP session, 30 mn = 30*60*1000 = 1800000 ms